domains/saumya.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "27Saumya",
+        "email": "patelsaumya767@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=saumya.is-a.dev,2235778835d5522c5d91"
```

Comment:
We do not support Vercel.

Reply:
oh no then, you may close this PR, i cant host elsewhere as of now